client/src/components/reader/Annotation/Popup/Wrapper.js

```diff
@@ -87,6 +107,9 @@ export default class AnnotationPopup extends Component {
     const halfPopupWidth = popupWidth / 2;
     const clientX = get(selectionClickEvent, 'clientX');
     const clientY = get(selectionClickEvent, 'clientY');
+    // const documentHeight = Math.max( document.body.scrollHeight, document.body.offsetHeight,
```

Comment:
Also, could you remove the comment at line 115 that I mentioned in slack? Thanks!